cloud-sql/postgres/appsettings.json

```diff
@@ -15,7 +15,14 @@
   "CloudSQL": {
     "Database": "PostgreSQL",
     "ConnectionString": "Uid=aspnetuser;Pwd=;Host=cloudsql;Database=votes"
+    // Uncomment and use the ConnectionString below when deploying to Cloud Run.
```

Comment:
I believe that comments are not allowed in JSON,  even if some parses do accetp it. And the readme seems to cover these already.